hicommonwealth/commonwealth
# 5726-modelsSemantics

## Link to Issue
Closes: #5726 

## Description of Changes
- Changed instances of `chain` and `chainId` to `community` and `communityId`
- updating everything within the `/models` folder

## Test Plan
- Create a user 
- Ensure notifications work as intended 
- Community Roles
- Ensure account info renders correctly
